R/forecast.R

```diff
@@ -459,27 +459,34 @@ is_forecast.default <- function(x, ...) {
 #' @export
 #' @rdname is_forecast
 #' @keywords check-forecasts
-is_forecast.forecast_sample <- function(x, ...) {
-  inherits(x, "forecast_sample")
+is_forecast.forecast <- function(x, ...) {
+  inherits(x, "forecast")
+}
```

Comment:
I think this should get the same treatment as the other functions. Due to the way S3 dispatch works, the only objects that will be forwarded here are `forecast` object, by construction.

I think this should be a normal function `is_forecast()`, an not an S3 method.

Reply:
Updated. The latest commits
- turn this S3 method into a normal function
- slightly update the documentation